stdlib/public/core/FloatingPoint.swift.gyb

```diff
@@ -676,13 +727,46 @@ extension FloatingPoint {
   }
 
   @_transparent
+  public func rounded(_ rule: FloatingPointRoundingRule) -> Self {
+    var lhs = self
+    lhs.round(rule)
+    return lhs
+  }
+
+  /// Returns `self` rounded to the closest integral value.  If `self` is
+  /// exactly halfway between two integers (e.g. 1.5), the integral value
+  /// with greater magnitude (2.0 in this example) is returned.
+  ///
+  /// Other rounding modes can be explicitly specified with
+  /// `.rounded(_: FloatingPointRoundingRule)`.
+  @_transparent
+  public func rounded() -> Self {
+    return rounded(.toNearestOrAwayFromZero)
+  }
+
+  /// Rounds `self` to the closest integral value.  If `self` is exactly
+  /// halfway between two integers (e.g. 1.5), the integral value with
+  /// greater magnitude is selected.
+  ///
+  /// Other rounding modes can be explicitly specified with
+  /// `.round(_: FloatingPointRoundingRule)`.
+  @_transparent
+  public mutating func round() {
+    round(.toNearestOrAwayFromZero)
+  }
+
+  @_transparent
+  public var nextDown: Self {
+    return -(-self).nextUp
```

Comment:
It's a default to simplify the implementation burden on types that want to conform.